packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -17,9 +17,9 @@ This guide is an overview of building your own user interface using the embedded
 
 ## Sign-in flow
 
-Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of *authenticators*, or ways of verifying the identity of a user, and assigns them to apps, people, groups, and more. Policies also configure whether an authenticator is required or optional, as well as the minimum number of authenticators required for a successful sign-on. Many flows require multiple *factors* (multifactor authentication), the category of an authenticator. Factors include biometrics, such as a fingerprint, knowledge, such as a password, and more. This results in a many of possible combinations of the different authenticators, the steps in the sign-in flow.
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of *authenticators*, or ways of verifying the identity of a user, and assigns them to apps, people, groups, and more. Policies also configure whether an authenticator is required or optional, as well as the minimum number of authenticators required for a successful sign-in. Responding to each authenticator is a step in the sign-in flow. Many flows require multiple *factors* (multifactor authentication), the category of an authenticator. Factors include: biometrics, such as a fingerprint; knowledge, such as a password; and more. This effectively results in an infinite number of combinations of authenticators and the order in which they're presented.
```

Comment:
Style guide would say never use italics. Please just use bold. On an additional note, markdown here sees _this_ and *this* both as italics rather than __this__ and **this** which are bold. So if you were trying to mix things up, it didn't work.

Also, an authenticator is the same as an authentication factor in this paragraph. The authentication factor type is biometric, knowledge, possession, etc

```suggestion
Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates **policies**. Each policy sets different mixes of **authenticators**, or ways of verifying the identity of a user, and assigns them to apps, groups, and more. Policies also configure if an authenticator is required or optional, and the minimum number of authenticators required to sign in successfully. Responding to each authenticator is a step in the sign-in flow. Many flows require multiple *factors* (multifactor authentication), the category of an authenticator. Factors include: biometrics, such as a fingerprint; knowledge, such as a password; and more. This effectively results in an infinite number of combinations of authenticators and the order in which they're presented.
```

Reply:
I reorganized this by starting with authenticators and working up to policies. I also split the paragraph in 2 as it was a bit lengthy. Is this OK?

> Okta supports many different __authenticators__ for identifying a user during the sign-in flow, such as a password or fingerprint. Responding to each authenticator is a step in the sign-in flow. Many flows require multiple __factors__ (multifactor authentication), the type of an authenticator. For example, a fingerprint is a biometric factor and a USB Token is a possession factor.
> 
> An Okta org administrator creates __policies__, different mixes of authenticators, and assigns them to apps, people, groups, and more. Policies also configure whether an authenticator is required or optional, as well as the minimum number of authenticators required for a successful sign-in. This effectively results in an infinite number of combinations of authenticators and the order in which they're presented.
>

Reply:
How about this change:
> Some flows can require multiple __factors__ (multifactor authentication). A factor is the type of an authenticator, such as a fingerprint for a biometric factor and a USB Token for a possession factor.